src/uct/ib/base/ib_device.h

```diff
@@ -80,6 +80,14 @@ typedef enum uct_ib_roce_version {
 } uct_ib_roce_version_t;
 
 
+typedef enum {
+    UCT_IB_LAG_QUEUE_AFFINITY = 0x0, /* QP are associated with port affinity */
+    UCT_IB_LAG_PORT_SELECT_FT = 0x1, /* packets go to egress port through FT */
+    UCT_IB_LAG_MULTI_PORT_ESW = 0x2, /* FDB select packet flow egress port */
+    UCT_IB_LAG_INVALID_MODE   = 0xFF
+} uct_ib_port_select_mode_t;
```

Comment:
need to move these to ib_mlx5_ifc.h and add "MXL5" in the name

Reply:
Done